src/core/services/interaction/interaction.js

```diff
@@ -0,0 +1,68 @@
+angular
+  .module('material.core.interaction', [])
+  .service('$mdInteraction', MdInteractionService);
+
+function MdInteractionService($timeout) {
```

Comment:
I don't think this service should be documented, it's not really generic and there aren't much use cases for a user to use that service. It's mostly a service which helps us with the accessibility for the components. But if you wan't me to document it, it won't be a problem.